features/step_definitions/env.rb

```diff
@@ -76,3 +76,13 @@
   end
 end
 
+Given(/^I have a single node cluster$/) do
```

Comment:
Perhaps `Given(/^the cluster is Single Node OpenShift$/)`

The current name make me think it's another cluster beside the current using one.

Reply:
Have updated, could you please help check ? Thanks !!